find_courses.py

```diff
@@ -21,17 +21,17 @@ def main():
     is_running = True
 
     while is_running:
-        print("\n\n--------WASHU TRANSFER COURSE FINDER--------")
+        print(consts.TITLE_MESSAGE)
 
         #get zip code from user
-        zip = raw_input("\n\nTo find colleges offering courses near you, enter your zip code: ")
+        zip = raw_input(consts.ZIP_MESSAGE)
```

Comment:
we need to stick with this format of imported constants throughout the file.
we want constants.TITLE_MESSAGE, not TITLE_MESSAGE